core/deployment/src/main/resources/default_banner.txt

```diff
@@ -1,4 +1,8 @@
-__  ____  __  _____   ___  __ ____  ______ 
- --/ __ \/ / / / _ | / _ \/ //_/ / / / __/ 
- -/ /_/ / /_/ / __ |/ , _/ ,< / /_/ /\ \   
---\___\_\____/_/ |_/_/|_/_/|_|\____/___/   
\ No newline at end of file
+    ████    ██   ██     ███     ██████   ██   ██  ██   ██   █████
+   ██░░██  ░██  ░██    ░███    ░██░░░██ ░██  ██  ░██  ░██  ██░░░██
+  ██  ░░██ ░██  ░██    ██░██   ░██  ░██ ░██ ██   ░██  ░██ ░██  ░░
+ ░██   ░██ ░██  ░██   ░██░██   ░██████  ░█████   ░██  ░██ ░░█████
+ ░██   ░██ ░██  ░██   ███████  ░██░██   ░██░░██  ░██  ░██  ░░░░░██
+ ░░██  ██  ░██  ░██  ░██░░░██  ░██░░██  ░██ ░░██ ░██  ░██  ██  ░██
+  ░░███░██  ░█████   ██   ░░██ ░██ ░░██ ░██  ░░█  ░█████  ░░█████
+   ░░░ ░░    ░░░░   ░░     ░░  ░░   ░░  ░░    ░    ░░░░    ░░░░░
```

Comment:
Latest idea. Only uses three characters, all of which appear in Latin-1. It's tall though, 8 characters.